ios/FluentUI/Shimmer/ShimmerView.swift

```diff
@@ -165,13 +170,19 @@ open class ShimmerView: UIView {
 
 	/// Update the frame of each layer covering views in the containerView
 	func updateViewCoverLayers() {
-		guard let containerView = containerView else {
-			return
-		}
+		let viewToCover = containerView ?? self
 
 		viewCoverLayers.forEach { $0.removeFromSuperlayer() }
 
-		let subviews = Set(containerView.subviews).subtracting(Set(excludedViews))
+		let subviews: Set<UIView> = {
+			if shimmersLeafViews {
+				var leaves = [UIView]()
+				searchLeaves(in: viewToCover, output: &leaves)
+				return Set(leaves).subtracting(Set(excludedViews))
```

Comment:
Shouldn't have early returns

Reply:
This is not a return of the entire function. This is a closure that lets me evaluate `subviews` as a `let` instead of a var